test/hls/hls_parser_unit.js

```diff
@@ -4582,6 +4582,33 @@ describe('HlsParser', () => {
     expect(actualManifest.presentationTimeline.getDuration()).toBe(5);
   });
 
+  it('parses #EXT-X-BITRATE', async () => {
+    const media = [
+      '#EXTM3U\n',
+      '#EXT-X-PLAYLIST-TYPE:VOD\n',
+      '#EXT-X-MAP:URI="init.mp4",BYTERANGE="616@0"\n',
+      '#EXTINF:5,\n',
+      '#EXT-X-BYTERANGE:121090@616\n',
+      '#EXT-X-BITRATE:385\n',
```

Comment:
The test needs to cover multiple segments, and EXT-X-BITRATE tags that cover multiple segments each.

Reply:
Done!